<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MicroSD_explorer</class>
 <widget class="QWidget" name="MicroSD_explorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>282</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>282</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MicroSD Card</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/icono_microSD.png</normaloff>:/icons/icono_microSD.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>258</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Canciones ubicadas en la MicroSD Card</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_canciones"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_agregar">
      <property name="text">
       <string>Agregar a lista actual</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ActualizarLista">
      <property name="text">
       <string>Actualizar Lista</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
